_posts/2018-01-19-event-lineup.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: "The 2018 Rust Event Lineup"
+author: Rust Community
+description: "Multitude of Rust event are happening this year; join us at one near you!"
+---
+
+Every year there are multiple Rust events around the world, bringing together the community.
+
+### February 1-3: Mozilla Rust Roadshow Brasil 2018
+
+The [Rust Roadshow][roadshow] is a series of workshops for Rust.
+Learn Rust with the community in several cities across Brazil.
+
+Several workshops are already scheduled for the beginning of February, with hopefully more to come.
+Find all information on [the Roadshow website][roadshow].
+
+[roadshow]: https://rust-br.github.io/2018-roadshow/
+
+### February 3-4: FOSDEM - Rust Dev Room
+
+After a very successful [Birds of a Feather][bof] session for Rust last year at FOSDEM,
+this year will have the first edition of the [Rust Dev Room][fosdem-rust].
+Co-located with [FOSDEM 2018][fosdem], the annual conference about free and open source software,
+it will bring together the Rust community in **Brussel, Belgium**.
```

Comment:
Nit: I believe it's Brussel**s** in English (I'm prone to making that mistake myself)

Reply:
You are right, fixed.